cpop_bot/command_handler.py

```diff
@@ -16,7 +16,7 @@ async def handle(self, bot_username, message: types.Message):
         elif message.text == "/start":
             await self.__handle_start(message)
         else:
-            logger.debug("USER TYPED UNKNOWN COMMAND " + message.text)
+            logger.debug("USER TYPED UNKNOWN COMMAND %s", message.text)
```

Comment:
is using %s a better practice in this, or why did you decide to change it

Reply:
it's interesting because this way of making an output
```
"USER TYPED UNKNOWN COMMAND %s", message.text
```
is actually used in **C**
Alright if pylint says it's better then i guess it should be right